src/python/bot/tasks/fuzz_task.py

```diff
@@ -419,6 +419,28 @@ def _track_testcase_run_result(fuzzer, job_type, new_crash_count,
   })
 
 
+def _last_sync_time(sync_file_path):
+  """Read and parse the last sync file for the GCS corpus."""
+  if not os.path.exists(sync_file_path):
+    return None
+
+  file_contents = utils.read_data_from_file(sync_file_path, eval_data=False)
```

Comment:
nit: why not just let lines 432 onwards handle this case also?

Reply:
My understanding is that empty file is ok (therefore just a warning), while corrupted means we have a problem and an error should be reported. My initial patch was to log a message like "either malformed or empty" and Abhishek suggested to catch the failure and separate these